plugin/evm/block.go

```diff
@@ -85,9 +85,11 @@ func (b *Block) Verify() error {
 	// Only enforce a minimum fee when bootstrapping has finished
 	if b.vm.ctx.IsBootstrapped() {
 		// Ensure the minimum gas price is paid for every transaction
+		timestamp := b.ethBlock.Header().Time
+		minGasPrice := b.vm.minGasPrice.GetMin(new(big.Int).SetUint64(timestamp))
```

Comment:
change this to a function on the block to make it more compatible with future ways we want to play around with gas prices